pkgs/top-level/all-packages.nix

```diff
@@ -31125,9 +31125,7 @@ with pkgs;
     withConplay = false;
   };
 
-  mpc-cli = callPackage ../applications/audio/mpc {
-    inherit (python3Packages) sphinx;
-  };
+  mpc-cli = mpc;
```

Comment:
I think this should just go into `aliases.nix`?

Reply:
Hum, OK.